docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -33,9 +33,23 @@ If you are [renting your rover](https://app.viam.com/try), we recommend that you
 Be aware that if you are running out of time during your rental, you can [extend your rover rental](/try-viam/reserve-a-rover/#extend-your-reservation) as long as there are no other reservations.
 {{< /alert >}}
 
+## Enable the cameras
+
+Before configuring color detection, enable the rover's camera to get a better sense of what it perceives as it performs color classification.
+
+* If you are running this tutorial with a [rented Viam Rover](https://app.viam.com/try), enable both provided cameras: the front-facing camera and the overhead cam.
+  In the `viam_base` component panel under the **Control** tab, enable both the `cam` for the front-facing camera and the `overhead-cam:overheadcam` for an overhead view of your rover.
+
+  ![Screenshot of the viam_base component panel showing both the 'cam' and 'overheadcam' camera feeds enabled.](try-viam/try-viam/enable-both-cameras.png)
+
+  You can also view and control the camera streams from the [individual camera component panels](/try-viam/try-viam-tutorial/#camera-control).
+
+* If you are running this tutorial on [your own Viam Rover](/try-viam/rover-resources/), enable the front facing camera.
+  If you are using the `ViamRover` [fragment](/try-viam/rover-resources/rover-tutorial-fragments/) with your rover, the front facing camera is named `cam` and can be enabled in the `viam_base` component panel under the **Control** tab.
+
 ## Add the vision service to detect a color
 
-This tutorial uses the hex color #7a4f5c (a reddish color).
+This tutorial uses the hex color `#7a4f5c` (a reddish color).
 
 **Hex color #7a4f5c**: {{<imgproc src="/tutorials/try-viam-color-detection/7a4f5c.png" resize="300x" declaredimensions=true alt="A color swatch for the color that you will be detecting with your color detector. It's a reddish, maroon color.">}}
```

Comment:
```suggestion
**Hex color #7a4f5c**: {{<imgproc src="/tutorials/try-viam-color-detection/7a4f5c.png" resize="200x" declaredimensions=true alt="A color swatch for the color that you will be detecting with your color detector. It's a reddish, maroon color.">}}
```
[opt out of diff nit] I just think this box looks oddly big, almost like a big redaction